Make osn-tests easier to run locally and in appveyor

pdamasceno opened the pull request:
There is one change to the path of obs-studio-node in imports of all tests that had to be made in order to make it all work

prsdamasceno committed:
Make osn-tests easier to run locally and on appveyor

prsdamasceno committed:
Add missing tsconfig.json file

prsdamasceno committed:
Remove local path from index.ts

EddyGharbi commented on package.json:
```diff
@@ -18,14 +18,27 @@
-    "build:docs": "typedoc --out docs"
+    "build:docs": "typedoc --out docs",
+    "local:config": "yarn install && git submodule update --init --recursive --force && cmake -Bbuild -H. -G\"Visual Studio 15 2017\" -A x64 -DCMAKE_INSTALL_PREFIX=\"./obs-studio-node\"",
+    "local:build": "cmake --build build --target install",
```
--config RelWithDebInfo/Debug is missing here.

pdamasceno replied:
It uses Debug by default if --config is not used. Still want me to add it there?

EddyGharbi replied:
I would make it explicit, doesn't really hurt and prevent some potential mistakes.

pdamasceno replied:
Made it explicit in the latest commit

prsdamasceno committed:
Add Debug flag to local build script

* Removed clean script because git clean can prevent some node_module
folders from being removed

prsdamasceno committed:
Remove index.ts

This file is generated by CMake, there is no need to track it in the
repository
